portfolio/src/main/webapp/request.js

```diff
@@ -55,7 +55,8 @@ function onClickStartScheduling() {
   const scheduleRequest = createScheduleRequestFromDom();
   // Create the request to send to the server using the data we collected from
   // the web form.
-  fetchScheduledTasksFromServlet(scheduleRequest).then((scheduledTaskArray) => {
+  fetchScheduledTasksFromServlet().then((scheduledTaskArray) => {
+    scheduledTasks = scheduledTaskArray;
```

Comment:
I think I mentioned this before, but do we need this scheduledTasks variable? Can it diverge from the DOM elements? It might be safer/simpler to treat DOM elements as the source of truth.